config_manager.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	MinimumRefreshInterval = (time.Duration(10) * time.Second)
+	MinimumRefreshInterval = RefreshInterval(1 * time.Second)
```

Comment:
Probably, yes. Could we go for a 10 second default and leave this as the minimum for those that are only monitoring a few files?